src/Helpers/PriceConversionHelper.cs

```diff
@@ -26,7 +26,7 @@ public static decimal GetBtcToUsdPrice()
         try
         {
             JsonDocument document = JsonDocument.Parse(response.Content);
-            btcPrice = document.RootElement[0].GetProperty("current_price").GetDecimal();
+            btcPrice = document.RootElement.GetProperty("current_price").GetDecimal();
```

Comment:
What about this change?

Reply:
It was my error. When coingeko fails, instead of throwing, it returns an "Object" with an error message. Now I check if it is an array before getting the price